src/modules/auth/services/auth.service.ts

```
@@ -0,0 +1,53 @@
+    const newUser: User = {
+      id: this.nextId++,
+      username,
+      password: hashedPassword,
+    };
+
+    this.users.push(newUser);
+    return newUser;
+  }
+
+  async login(username: string, password: string): Promise<User> {
+    const user = this.users.find(user => user.username === username);
```

ESLint

[error] Replace `user` with `(user)`

(prettier/prettier)